js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
```

Comment:
Yeah, in this case, I'm okay with it. I did a bit of digging into de-/pluralization libraries, and found out that it's a whole loaded mess. This is clearly insufficient for broader use, but unless you know of a middle ground, something like moment.js for plurals, I think this'll have to do.